_source/logzio_collections/_log-sources/aws-cross-account.md

```diff
@@ -0,0 +1,223 @@
+| `SendingAccounts`        | Comma-delimited list (**no spaces**) of account numbers of the accounts you'd like to send logs froms. The numbers should be specified as follows: `"1234","5678","9012"`                                                                                                                                                                  | Required           |
+
+
+
+##### Configure stack options
+
+If required, specify the **Key** and **Value** parameters for the **Tags** and select **Next**.
+
+##### Review the stack
+
+Confirm that you acknowledge that AWS CloudFormation might create IAM resources, IAM resources with custom names, may require `CAPABILITY_AUTO_EXPAND` and select **Create stack**.
+
+
+##### Create a stack to deploy destinations in the landing account on each region you need to send logs from
+
+Set the region of your AWS account to `us-east-1` and launch the stach by clicking the button below:
+
+[![Deploy to AWS](https://dytvr9ot2sszz.cloudfront.net/logz-docs/lights/LightS-button.png)](https://console.aws.amazon.com/cloudformation/home?region=us-east-1#/stacks/new?stackName=aws-cross-account-destination&templateURL=https://integrations-testing.s3.amazonaws.com/cross-accounts/0.0.1/sam-template-destination.yaml)
+
+
+##### Specify the stack template
+
+Keep the default settings in the **Create stack** screen and select **Next**.
+
+##### Specify the stack details
+
+Specify the stack details as per the table below and select **Next**.
+
+| Parameter          | Description                                                                                                                                                 |
+|--------------------|-------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `RoleArn`          | The ARN of the **Role** created in the main stack. You can find it in the main stack's **Outputs** tab, under **LogzioCrossAccountRole**                        |
+| `KinesisStreamArn` | The ARN of the **Kinesis Stream** created in the main stack. You can find it in the main stack's **Outputs** tab, under **LogzioCrossAccountKinesisStreamArn**. |
+| `SendingAccounts`        | Comma-delimited list (**no spaces**) of account numbers of the accounts you'd like to send logs froms. The numbers should be specified as follows: `"1234","5678","9012"` | 
+
+##### Configure stack options
+
+If required, specify the **Key** and **Value** parameters for the **Tags** and select **Next**.
+
+##### Review the stack
+
+Confirm that you acknowledge that AWS CloudFormation might create IAM resources, IAM resources with custom names, may require `CAPABILITY_AUTO_EXPAND` and select **Create stack**.
+
```

Comment:
Can we add a sub-header so that it will be clearer that step 11 is in the sending accounts?